src/game-init.cc

```diff
@@ -464,37 +469,60 @@ GameInitBox::handle_click_left(int cx, int cy) {
   return true;
 }
 
+unsigned int
+GameInitBox::get_next_character(unsigned int player_index) {
+  bool in_use = false;
+  PPlayerInfo player = mission->get_player(player_index);
+  unsigned int next = player->get_face();
+  do {
+    next = (next + 1) % 14;
+    next = std::max(1u, next);
+        
+    /* Check that face is not already in use by another player */
```

Comment:
Think you could have AI sharing characters.

Reply:
Unfortunately, can't yet.